src/plugins/guided_onboarding/common/constants.ts

```diff
@@ -10,3 +10,5 @@ export const PLUGIN_ID = 'guidedOnboarding';
 export const PLUGIN_NAME = 'guidedOnboarding';
 
 export const API_BASE_PATH = '/api/guided_onboarding';
```

Comment:
This API really should be `/internal/guided_onboarding`. I can't think of any reason why a user would need to use this API directly, and by using the `/api` path prefix, we are indicating it is public, meaning you can't make any breaking changes to it without approvals, a long deprecation period, etc.

Reply:
Agree @lukeelmers, I opened this [issue](https://github.com/elastic/kibana/issues/155611) to change the route's prefix. Our team was not aware of this convention when setting up the plugin. 
What do you think, do we have to handle the prefix issue as a breaking change or can we update it in the next minor? The plugin was first released in 8.7

Reply:
Since it was introduced recently & is undocumented (and also something folks would be unlikely to discover and get much utility out of), then IMO if you fix it quickly, you are probably safe to change it, treat this as a bug, and document it in the release notes.